src/Common/LoggingConstants.cs

```diff
@@ -12,12 +13,16 @@ public static class LoggingConstants
         public const string ApplicationInsightsInstrumentationKeyEnvironmentVariableName
             = "ORYX_AI_INSTRUMENTATION_KEY";
 
-        public const string AppServiceAppNameEnvironmentVariableName = "APPSETTING_WEBSITE_SITE_NAME";
-        public const string ContainerRegistryAppNameEnvironmentVariableName = "REGISTRY_NAME";
         public const string DefaultOperationName = ".oryx";
 
         public const string DefaultLogPath = "/tmp/oryx.log";
 
+        public static readonly IDictionary<string, string> OperationNameSourceEnvVars = new Dictionary<string, string>
```

Comment:
We need changes in runtime startup scripts too?

Reply:
yes, in a separate PR

Reply:
sound good